proposals/incubation/kyverno.md

```diff
@@ -0,0 +1,160 @@
+# Kyverno for incubating 
+
+## Table of Contents
+
+- [Background](#background)
+- [About Kyverno](#about-kyverno)
+- [Progress Since Joining CNCF](#progress-since-sandbox)
+- [Incubation Stage Requirements](#incubation-stage-requirements)
+- [Statement on alignment with the CNCF mission](#statement-on-alignment-with-the-cncf-mission)
+- [Comparison with other CNCF Policy Management projects](#comparison-with-other-cncf-chaos-engineering-projects)
+- [Roadmap](#roadmap)
+
+## Background
+
+Kubernetes has a declarative configuration management system that allows users to specify the desired state of resources which Kubernetes controllers continuously reconcile with the current system state. Kyverno uses Kubernetes custom resource definitions for cluster-wide and namespaced policies, and the [Kubernetes Policy WG policy report](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report) for reporting policy results.
+
+Kyverno was accepted as a CNCF Sandbox project on [November 9th, 2020](https://docs.google.com/spreadsheets/d/1Nnh_usr0tSZxaUpxTusqeIqKxMmvuEViRkyO9e_Do40/edit#gid=1136111842).
+
+## About Kyverno
+
+Kyverno is a policy engine designed for Kubernetes. With Kyverno, policies are managed as Kubernetes resources and no new language is required to write policies. This allows using familiar tools such as `kubectl`, `git`, and `kustomize` to manage policies. Kyverno policies can validate, mutate, and generate Kubernetes resources. The Kyverno CLI can be used to test policies and validate resources as part of a CI/CD pipeline.
+
+### High-level Architecture
+
+Kyverno comprises of four major components, each component is described below:
+
+<img src="https://kyverno.io/images/kyverno-architecture.png" alt="Kyverno Architecture" width="80%"/>
+</br></br>
+
+**Webhook**: The `Webhook` component registers as a validating and mutating admission webhook and receives `AdmissionReview` requests from the API server to validate and mutate configuration changes, based on policies. Users can configure which namespaces and resources the webhooks will receive via command line options or the ConfigMap.
```

Comment:
In this section where the components are described, suggest avoid using code formatting (wrapping text in back ticks) unless it specifically refers to something to be typed or the formal name of a resource (ex., `GenerateRequest`).